docs/modules/security/pages/client-authorization.adoc

```diff
@@ -191,6 +191,8 @@ The following is the list of client permissions that can be configured on the me
 === All permissions
 
 The `<all-permissions>`` permission grants clients access to all data and features.
+The client specified in the following code example is used by Hazelcast Management Center when it connects to clusters.
+To learn more about this client, see xref:{page-latest-supported-mc}@management-center:ROOT:connecting-members.adoc[].
```

Comment:
This doesn't have link text in the preview build. The existing link looks fine on the live site so this should too, but please check once this update goes live (a few minutes after the changes are merged).